app/registries/foreman/plugin.rb

```diff
@@ -39,6 +40,7 @@ class PluginRequirementError < Foreman::Exception; end
   class Plugin
     @registered_plugins = {}
     @tests_to_skip = {}
+    @importer_registry = Plugin::ImporterRegistry.new
```

Comment:
Now that I think more about it, it makes sense to me to call it `fact_importer_registry`/`FactImporterRegistry` just so it's clear what is being imported - there are other things that could be imported into foreman. Sorry for not thinking about it earlier.

Reply:
No problem, i will make the changes right away! 
@ShimShtein told me about it and we thought it makes sense to have it called as `FactImporterRegistry` but it becomes a little bigger for a class name. But now that we are all fine with the name, I will go with it :) thanks @tbrisker